client/src/app/app.component.ts

```diff
@@ -88,10 +99,24 @@ export class AppComponent {
                 filter(s => s),
                 auditTime(1000)
             )
-            .pipe(take(2))
-            .subscribe(() => {
-                spinnerService.setVisibility(false);
+            .subscribe(stable => {
+                // check the stable state only once.
+                if (!this.isStable && stable) {
+                    this.isStable = true;
+                    this.checkConnectionProgress();
+                }
             });
+        // subscribe, to get the user.
+        operator.getViewUserObservable().subscribe(user => {
```

Comment:
Please be careful here! I guess, that if the anonymous is enabled, `user` will always stay `null`... Please try this out. If so, I can give you hints, where to get these information (users loaded finished) from somewhere else.

Reply:
You're right, the user will always be `undefined`... Can you give me any hints for this, please? Thanks!

Reply:
**Update**: I have pushed a second commit - how do you think about my solution for this?